n2v/inverter.py

```diff
@@ -234,14 +238,14 @@ def generate_components(self, potential_components):
         J, _ = self.form_jk( self.ct[0], self.ct[1] )
         self.Hartree_a, self.Hartree_b = J[0], J[1]
 
-        if "fermi_amaldi" in potential_components:
-            v_fa = (-1/N) * (J[0] + J[1])
+        if "fermi_amaldi" in guide_potential_components:
+            v_fa = (1-1/N) * (J[0] + J[1])
 
             self.va += v_fa
             self.vb += v_fa
 
-        if "svwn" in potential_components or "pbe" in potential_components:
-            if "svwn" in potential_components:
+        if "svwn" in guide_potential_components or "pbe" in guide_potential_components:
```

Comment:
I added this very early on the code before acknowledging that we wouldn't easily be able to plot anything other than LDA. I'd be fine with removing the "pbe" option. Otherwise, we can think about adding the rest of the elements for vxc later.